File: Lv.0/배열 두배 만들기.js
/* 정수 배열 numbers가 매개변수로 주어집니다.
numbers의 각 원소에 두배한 원소를 가진 배열을 return하도록 solution 함수를 완성해주세요.
 */

function solution(numbers) {
  return numbers.map((number) => number * 2);
}
console.log(solution([1, 2, 3, 4]));

//다른사람 풀이

function solution2(numbers) {
  return numbers.reduce((a, b) => [...a, b * 2], []);
}

File: Lv.0 알고리즘/배열두배만들기.py
""" 정수 배열 numbers가 매개변수로 주어집니다.
numbers의 각 원소에 두배한 원소를 가진 배열을 return하도록 solution 함수를 완성해주세요.
 """
# 리스트 컴프리헨션 방식

def solution(numbers):
    return [num*2 for num in numbers]